src/components/page/Header/index.tsx

```diff
@@ -288,6 +292,20 @@ export class HeaderComponent extends React.Component<HeaderProps, State> {
                         </DropdownItem>
                       </div>
                     )}
+
+                    {ENABLE_MDA_LITE && (
```

Comment:
Does the enable/disable also work at the route level? i.e Can you access the route if the menu link is disabled?

Reply:
Yeah the link will be accessible. Fixing that.

Reply:
@kelvin-muchiri this is affecting all routes in reveal and have created an issue [here](https://github.com/onaio/reveal-frontend/issues/1551) to address it.